docs/hardware/4-nrf91/2-quickstart/3-flash-firmware/2-bootloader.md

```diff
@@ -0,0 +1,123 @@
+---
+id: bootloader
+title: Flash via USB Bootloader
+---
+
+## Installing Tools
+
+Flashing via USB bootloader requires the `newtmgr` tool.
+
+### Installing Newt Manager (for bootloading)
+
+We recommend you use [Jared Wolff's fork of newtmgr](https://github.com/circuitdojo/mynewt-newtmgr) which includes customization for higher-speed transfers on MCUboot implementations that support it. This tool depends on the Go language so we will also install that.
+
+1. Install the Go Language
+
+    :::note
+    You can install the Go language with `sudo apt install golang-go`, but some package managers have older versions. The following process will install the newest version.
+    :::
+
+    Visit [the Go language download page](https://go.dev/dl/) and download the latest version. Install the package:
+
+    ```bash
+    cd ~
+    tar -xvf go1.17.6.linux-amd64.tar.gz --directory ~/.
+    nano ~/.profile
+    ```
+
+    Add the following lines to the end of this file:
+
+    ```bash
+    export GOPATH=$HOME/go
+    export PATH=$PATH:$GOPATH/bin
+    ```
+
+    Save the file and load the changes:
+
+    ```bash
+    source ~/.profile
+    ```
+
+2. Install Newt Manager (newtmgr)
+
+    ```bash
+    mkdir -p $GOPATH/src/mynewt.apache.org/newtmgr
+    git clone https://github.com/circuitdojo/mynewt-newtmgr.git $GOPATH/src/mynewt.apache.org/newtmgr
+    cd $GOPATH/src/mynewt.apache.org/newtmgr/newtmgr
+    GO111MODULE=on go build
+    go install
+    ```
+
+## Flashing Firmware
+
+The nRF9160 can be programmed via USB using the MCUboot bootloader. The process depends on three steps: compiling your application for MCUboot, placing the target board in Device Firmware Upgrade (DFU) mode, and using a device management tool to write the firmware binary to the target.
+
+### Compile your code with MCUboot in mind
+
+1. Add the `CONFIG_BOOTLOADER_MCUBOOT` flag to the project's `prj.conf` file.
+
+    ```bash
+    CONFIG_BOOTLOADER_MCUBOOT=y
+    ```
+
+2. Build the project:
+
+    ```bash
+    cd ~/zephyr-nrf/modules/lib/golioth
+    #Flashing example for Circuit Dojo nRF9160 Feather
+    west build -b circuitdojo_feather_nrf9160_ns samples/basic/blinky -p
+    #Flashing example for Thiny:91
```

Comment:
Thingy91 is misspelled multiple times

Reply:
Thanks, I fixed all of these up in https://github.com/golioth/docs/pull/158/commits/f29c818c8b18d518915624383ebe5344242e7dfd#